### PR TITLE
leaks03 added

### PR DESCRIPTION
**Pull request by cloudziq**
two new decals

### CODE REVIEW

File: dziq/sharingan.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+setshader bumpspecdecal
+setshaderparam bumpblend 0.8
+setshaderparam specscale 0.2
+    texture decal "decals/dziq/sharingan"
```
Comment (Jigoku) [inline dziq/sharingan.cfg]: Isn't this a copyrighted emblem?

Comment (SniperGoth) [reply dziq/sharingan.cfg]: Yeah, it is, from naruto.

Comment (cloudziq) [reply dziq/sharingan.cfg]: yeah I rushed with this,  hard to find any info so I removed it
EDIT:  added `damaged03` instead

File: dziq/damaged03.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+setshader bumpdecal
+   texture decal "decals/dziq/damaged03.png"
+   texture n "decals/dziq/damaged03_n.png"
```
Comment (qreeves) [inline dziq/damaged03.cfg]: No newline at end of file.

File: dziq/leaks03.cfg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+setshader bumpdecal
+   texture decal "decals/dziq/leaks03.png"
+   texture n "decals/dziq/leaks03_n.png"
```
Comment (qreeves) [inline dziq/leaks03.cfg]: No newline at end of file.

**Review by qreeves — CHANGES_REQUESTED** [review]

File: dziq/leaks03.cfg (hunk 3 above)
Comment (cloudziq) [reply dziq/leaks03.cfg]: both fixed, thank you

**Pull request merged**